function-images/tests/chained-function-eventing/consumer/main.go

```diff
@@ -34,23 +34,30 @@ import (
 func callback(_ context.Context, event cloudevents.Event) (*cloudevents.Event, cloudevents.Result) {
 	var body GreetingEventBody
 	if err := event.DataAs(&body); err != nil {
-		log.Fatalf("Consumer error while extracting CloudEvent data: %s", err)
+		log.Fatalf("failed to extract CloudEvent data: %s", err)
 	}
 
-	log.Printf("Consumer received a GreetingEvent: name=`%s`", body.Name)
+	log.Printf("received a GreetingEvent: name=`%s`", body.Name)
 
-	return nil, nil
+	response := cloudevents.NewEvent("1.0")
```

Comment:
I wonder if there is a method to set all or several fields in one go. should be there

Reply:
Sadly there isn't: https://pkg.go.dev/github.com/cloudevents/sdk-go/v2@v2.4.1/event#Event